sdk/resourcemanager/Azure.ResourceManager/tests/Samples/AuxiliaryPoilcy.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.ResourceManager.Tests.Samples
+{
+    #region Snippet:Sample_Header_Policy
+    public class AuxiliaryPoilcy : HttpPipelineSynchronousPolicy
```

Comment:
Since this is only used below, maybe we could change the accessibility to `internal`

Reply:
Fixed